modules/app/src/main/resources/admin/widgets/stats/stats.html

```diff
@@ -1,40 +1,24 @@
 <widget id="widget-stats">
-    <div style="display: flex; justify-content: center">
-        <div style="padding: 10px; text-align: center">
-            <h1 style="color: #333;">
+    <link rel="stylesheet" href="{{stylesUri}}" type="text/css" media="all"/>
+
+    <div class="stats-container">
+        <div class="total-items-container">
+            <h1 class="counter">
                 {{totalItems}}
             </h1>
-            <div style="color: #777;">
+            <div class="label">
```

Comment:
Both `counter` and `label` are too generic class names. Just think about that other widgets can use the same too. It would be good if we could shadow all class names behind widget id, like `widget-stats-container`, `widget-stats-label` etc.